### ISSUE TITLE
viewer show nothing

### ISSUE THREAD

**Issue by kijongGil** [splintered-reality/py_trees_ros_viewer#33]
Hi, I'm Gilbert.

I'm developing behavior tree using py-trees-ros.

My environment is here:
* Linux Mint 19.3
* ROS2 Dashing
* py-trees-ros-viewer : release/0.1.x

I tried your quick start. 

```
sudo apt install ros-dashing-py-trees-ros-tutorials
sudo apt install ros-dashing-py-trees-ros-viewer

py-trees-tree-viewer
ros2 run py_trees_ros_tutorials tutorial-eight-dynamic-application-loading
```

But, viewer show nothing.
![Selection_002](https://user-images.githubusercontent.com/17861812/93157991-4542d280-f746-11ea-8650-52076b5a81a0.png)
Then, When I run the viewer, segmentation fault errors frequently occur.

Could you help me?

Thanks,
Gilbert

**Comment by stonier** [comment]
Ouch, `dashing` was really moving through gears. Everything stabilised in `eloquent` and `foxy`.

You *might* be running afoul of https://github.com/splintered-reality/py_trees_js/issues/141, which was fixed for `eloquent` (`py_trees_js v0.6`) ... However, in between all of those packages being very experimental (so much so I retracted official support for `py_trees v1.3` and left the last stable `1.x` release at `1.2`) and leaving `dashing behind` a long time ago, it could be a number of other things.

Any chance you can move to `eloquent` or `foxy` or are you tied to a robot still running on dashing? I'll be able to support you easily on these, especially the latter since it's a LTS release.

**Comment by kijongGil** [comment]
Thanks to reply, stonier.

I understand what the problem is. It's such a shame I can't use the viewer on dashing, but py-trees-ros pkgs are very helpful to me.

When I will move to foxy later, I will try again.

Thanks!
